.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue is requesting
+a new topic, please preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
+effort is spent writing a new topic. Please see our
+[Contributing Guide](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md)
+for details.
+
+## Issues with Existing Topics
+
+Please include a link to the topic. Describe the problem as specifically as you can. Consider:  
+
+- Is content incorrect or outdated?
+- Are we missing scenarios for the topic?
+- Do we need samples that explain the scenarios?
+- Would the explanation be clearer with images?
+
+## Requests for new Topics
+
+Tell us where this topic should go in the Table of Contents. If you're not sure, consider
+where you looked for this information before opening an
+issue.
+
+1. Write an abstract
+
+   - In one **short** paragraph, describe what this topic will cover.
+
+2. Fill in an outline
+
+    - Fill in as much as you can about how someone should cover the topic. We'll help review the outline and approve that before anyone writes a  topic.
+
+3. Suggest reviewers
+
+    - If you know who should be looking at this topic, please '@' mention them here. If you're not sure, just leave this blank. The core content team looks at new issues regularly.
```

Comment:
If you know someone who can provide feedback, use @<alias> to ask them to review.